website/content/docs/operations/key-management.mdx

```diff
@@ -7,13 +7,13 @@ description: Learn about the key management in Nomad.
 # Key Management
 
 Nomad servers maintain an encryption keyring used to encrypt [Variables][] and
-sign task [workload identities][]. The servers store key metadata in raft, but
-the encryption key material is stored in a separate file in the `keystore`
-subdirectory of the Nomad [data directory][]. These files have the extension
-`.nks.json`. The key material in each file is wrapped in a unique key encryption
-key (KEK) that is not shared between servers. Nomad servers can use an external
-KMS or Vault transit encryption to wrap the key material using the [`keyring`][]
-configuration block.
+sign task [workload identities][]. The servers encrypt these data encryption
+keys (DEK) and store the wrapped keys in Raft.
+
+The key encryption key (KEK) used to encrypt the DEK is controlled by the
+[`keyring`][] provider. When using an external KMS or Vault transit encryption
+providers, the KEK is securely stored outside of Nomad. For the default AEAD
+provider, the KEK is stored in cleartext in Raft.
 
```

Comment:
```suggestion
<Note>
- Default AED provider users should be aware that storing the Data Encryption Key (DEK) and Key Encryption Key (KEK) in cleartext in Raft may expose data in case of a breach.
</Note>
```

Reply:
Done. Reworded slightly because the DEK is never in cleartext.